ompi/mpi/fortran/use-mpi-f08/Makefile.am

```diff
@@ -539,13 +535,19 @@ endif
 #
 
 lib@OMPI_LIBMPI_NAME@_usempif08_la_LIBADD = \
-        profile/libmpi_usempif08_pmpi.la \
+        lib@OMPI_LIBMPI_NAME@_usempif08_profile.la \
         $(OMPI_MPIEXT_USEMPIF08_LIBS) \
         $(top_builddir)/ompi/mpi/fortran/mpif-h/lib@OMPI_LIBMPI_NAME@_mpifh.la \
         $(top_builddir)/ompi/lib@OMPI_LIBMPI_NAME@.la \
         mod/libusempif08_internal_modules.la \
         base/libusempif08_ccode.la
-lib@OMPI_LIBMPI_NAME@_usempif08_la_DEPENDENCIES = $(module_sentinel_files)
+#
```

Comment:
is this comment still relevant?

Reply:
I think so. I couldn't get it to build the profile library in the right order without this. I'm not sure why it's not detecting that dependency automatically.